### DISCUSSION TITLE
Rule Suppression

### DISCUSSION THREAD

**Discussion by pappajohndd** [Security-Onion-Solutions/securityonion#2482] [Unsupported Versions]
Is rule suppression broken or am I doing it wrong? Do I just append to the end of the file global.sls in a distributed deployment?
i.e. 
thresholding:
sids:
11223344:
      - suppress:
          gen_id: 1
          track: by_src
          ip: 10.10.3.0/24

Anytime I alter global.sls I get on error after running sudo salt somanager_manager state.apply idstools:
**somanager_manager:
Data failed to compile:
The function "state.highstate" is running as PID 244556 and was started at 2020, Dec 29 14:18:50.940850 with jid 20201229141850940850
ERROR: Minions returned with non-zero exit code**

If I restore the original global.sls the command runs with no error_

**Comment by dougburks** [comment]
Rule suppression should be working properly. Have you double-checked your syntax and your spacing?
https://docs.securityonion.net/en/2.3/managing-alerts.html#suppressions

**Comment by mtnsec** [comment]
I had a similar issue with thresholding and solved it by putting the threshold statements in the sensor's sls file instead of the manager's sls.

**Comment by abs0248** [comment]
Just make sure that not defining the specific SID twice or more in the pillar.And i think to let the salt making the threshold.conf you need the  command `sudo salt somanager_manager state.apply suricata`

**Reply by B3DTech** [reply]
So how do you create a suppression that contains multiple IP addresses/networks?

Would you add multiple ip: lines, or comma separate them?

ip: 10.85.128.5
ip: 10.85.128.6

or

ip: 10.85.128.5,10.85.128.6

**Reply by mtnsec** [reply]
I haven't tried anything like this, and I'm still super unsure about salt - but the pillars seem to be in YAML so I'd maybe try something like this:

ip: [10.85.128.5, 10.85.128.6]
